<?xml version="1.0"?>
<launch>

  <arg name="prefix"		default="biclops_"/>
  <arg name="publish_rate"	default="50"/>

  <!-- Spawn controller -->
  <node name="$(arg prefix)point_head_tracker"
        pkg="aist_controllers" type="point_head_tracker"
        output="screen" respawn="false">
    <rosparam file="$(dirname)/../config/point_head_tracker.yaml"
              subst_value="true"/>
    <remap from="/state" to="$(arg prefix)pos_joint_traj_controller/state"/>
  </node>

</launch>
